InterfaceParser/model/interface.py

```diff
@@ -0,0 +1,22 @@
+"""Interface
+
+"""
+
+from collections import OrderedDict
+
+
+class Interface:
+    """Interface.
+
+    :param enums: dictionary of enumerations
+    :param structs: dictionary of structures
+    :param functions: dictionary of functions
+    :param params: dictionary of interface parameters (name, version, etc.)
+
+    """
+
+    def __init__(self, enums=None, structs=None, functions=None, params=None):
```

Comment:
Does interface need to include param: name, version, minVersion, and date?

Reply:
after parsing attribute "params" will contain dictionary as follow:
params = {dict} {'version': '6.0.0', 'minVersion': '1.0', 'date': '2019-03-19'}
so "name" seems to be skipped, but as I aware it doesn't used anywhere.